hyperapp.d.ts

```diff
@@ -0,0 +1,149 @@
+declare module "hyperapp" {
+  export interface VNodeData {
+    [key: string]: any
```

Comment:
I think you can use the example I have from Picodom. We should absolutely avoid using `any` and prefer `{}` when possible.